jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/form/FormOverviewTests.java

```diff
@@ -0,0 +1,262 @@
+                .tag(TAG_NAME_INPUT);
+
+        formOverview.help.is()
+                .displayed()
+                .enabled()
+                .core()
+                .attr(ATTR_NAME_ID, ATTR_NAME_EMAIL_HELP)
+                .cssClass(CLASS_NAME_FORM_TEXT_TEXT_MUTED)
+                .text(emailHelp)
+                .value(emailHelp)
+                .tag(TAG_NAMR_SMALL);
+
+        formOverview.passwordLabel.is()
+                .displayed()
+                .enabled()
+                .core()
+                .value(passwordValue)
+                .text(passwordValue)
+                .attr(ATTR_NAME_FOR, ATTR_VALUE_EXAMPLE_INPUT_PASSWORD_1)
+                .tag(TAG_NAME_LABEL);
+
+        formOverview.password.is()
+                .displayed()
+                .enabled()
+                .core()
+                .cssClass(CLASS_NAME_FORM_CONTROL)
+                .attr(ATTR_NAME_TYPE, passwordValue.toLowerCase())
+                .attr(ATTR_NAME_ID, ATTR_VALUE_EXAMPLE_INPUT_PASSWORD_1)
+                .attr(ATTR_NAME_PLACEHOLDER, passwordValue)
+                .tag(TAG_NAME_INPUT);
+
+        formOverview.accept.is()
+                .displayed()
+                .enabled()
+                .core()
+                .cssClass(CLASS_NAME_FORM_CHECK_INPUT)
+                .attr(ATTR_NAME_TYPE, ATTR_VALUE_CHECKBOX)
+                .attr(ATTR_NAME_ID, ATTR_VALUE_EXAMPLE_CHECK_1)
+                .tag(TAG_NAME_INPUT);
+
+        formOverview.acceptLabel.is()
+                .displayed()
+                .enabled()
+                .core()
+                .text(acceptLabelText)
+                .value(acceptLabelText)
+                .cssClass(CLASS_NAME_FORM_CHECK_LABEL)
+                .attr(ATTR_NAME_FOR, ATTR_VALUE_EXAMPLE_CHECK_1)
+                .tag(TAG_NAME_LABEL);
+
+        formOverview.submit.is()
+                .displayed()
+                .enabled()
+                .core()
+                .cssClass(CLASS_NAME_BTN_BTN_PRIMARY)
+                .attr(ATTR_NAME_TYPE, ATTR_VALUE_SUBMIT)
+                .text(submitText)
+                .value(submitText)
+                .tag(TAG_NAME_BUTTON);
+    }
+
+    @Test
+    public void checkboxTests() {
+        formOverview.accept.check();
+        formOverview.accept.is().selected();
+        formOverview.accept.uncheck();
+        formOverview.accept.is().deselected();
+    }
+
+    @Test
+    public void fillTest() {
+        formOverview.fill(DEFAULT_CREDENTIALS);
+        formOverview.check(DEFAULT_CREDENTIALS);
+        checkContactFormSubmitted();
+    }
+
+    @Test(dataProvider = "fieldCheck",
+            expectedExceptions = {RuntimeException.class},
+            expectedExceptionsMessageRegExp = ".*Check form failed:.*")
+    public void fillNegativeTest(FormSignUp INCORRECT_CREDENTIALS) {
+
+        formOverview.fill(DEFAULT_CREDENTIALS);
+        formOverview.check(INCORRECT_CREDENTIALS);
+        checkContactFormSubmitted();
```

Comment:
This string is excess

Reply:
done

Reply:
I meen this string:
`checkContactFormSubmitted();`

It will not ever run, because your test must always throw exception at previous string.